generated/proto/commonpb/common.proto

```diff
@@ -0,0 +1,38 @@
+// Copyright (c) 2017 Uber Technologies, Inc.
```

Comment:
Talked offline gonna keep the package name and the directory name in sync to make IDE auto complete easier